website/www/site/content/en/documentation/programming-guide.md

```diff
@@ -7282,7 +7282,16 @@ To create an SDK wrapper for use in a Python pipeline, do the following:
 
 #### 13.1.2. Creating cross-language Python transforms
 
```

Comment:
Can you also update:

(1) "Including dependencies" section, to include information regarding "PythonExternalTransform.withExtraPackages" option.

(2) "Starting the expansion service" section to include "--pickle_library=cloudpickle" option. 

This should include

Reply:
Done.